### PR TITLE
Enable promotion workflow for SubM 2.6

### PR DESCRIPTION
**Pull request by dfarrell07**
See discussion:

https://coreos.slack.com/archives/CSZLMKPS5/p1663010291227569

Signed-off-by: Daniel Farrell <dfarrell@redhat.com>

### CODE REVIEW

**Comment by dfarrell07** [comment]
```
[~/go/src/openshift/release]$ make jobs                                                                                                                                                                                                                                                                                                                                                                                                                                            *[master]
false || docker pull registry.ci.openshift.org/ci/ci-operator-checkconfig:latest
Error response from daemon: Head "https://registry.ci.openshift.org/v2/ci/ci-operator-checkconfig/manifests/latest": unauthorized: authentication required
make: *** [Makefile:73: ci-operator-checkconfig] Error 1
```

That target has always "just worked" for me, not sure why it seems to need auth now.

File: ci-operator/config/stolostron/submariner-addon/stolostron-submariner-addon-main.yaml [hunk 1]
```diff
@@ -26,7 +26,6 @@ images:
       - registry.ci.openshift.org/stolostron/builder:go1.18-linux
   to: submariner-addon
 promotion:
-  disabled: true
```
Comment (tpantelis) [inline ci-operator/config/stolostron/submariner-addon/stolostron-submariner-addon-main.yaml:29]: From @gurnben, promotion should be disabled on main.

Comment (dfarrell07) [reply ci-operator/config/stolostron/submariner-addon/stolostron-submariner-addon-main.yaml]: Ok, fixed.

**Comment by dfarrell07** [comment]
These are typical errors, I think we see them every time.

**Pull request merged**